web/explorer/src/components/DescriptionPanel.vue

```diff
@@ -3,27 +3,45 @@
             <h1>
                 <img src="@/assets/images/logo-full.png" alt="Capa: identify program capabilities" />
-                <h3 class="font-medium" style="color: rgb(176, 26, 26)">capa: identify program capabilities</h3>
+                <h3 class="font-medium" style="color: rgb(37, 147, 215)">capa: identify program capabilities</h3>
             </h1>
         </div>
         <div>
             <p class="text-xl p-3 max-w-75rem" style="max-width: 75ch">
-                capa explorer web is a web-based tool for exploring the capabilities identified in a program. It can be
-                used to search and display the rule matches in different viewing modes.
+                capa Explorer Web is a web-based tool to explore the capabilities identified by capa. This tool allows
+                you to interactively browse and display capa results in multiple viewing modes.
             </p>
         </div>
         <div class="bg-blue-50 border-round p-3 pb-4 text-center max-w-75rem mt-4">
             <p class="text-700 text-sm m-0">New to capa? Follow these quick steps to get started:</p>
             <p class="text-700 text-sm mb-3"></p>
             <ol class="text-left text-sm pl-4 mb-3">
-                <li class="mb-2">Install capa: <code>pip install flare-capa</code></li>
-                <li class="mb-2">Generate JSON output: <code>capa -j /path/to/file > result.json</code></li>
-                <li>Upload the JSON file here to explore the results</li>
+                <li class="mb-2">
+                    <a href="https://github.com/mandiant/capa/blob/master/doc/installation.md" class="underline"
+                        >Install capa</a
+                    >, e.g.
+                    <ul>
+                        <li>
+                            download the latest
+                            <a href="https://github.com/mandiant/capa/releases" class="underline"
+                                >standalone executable release</a
+                            >
+                        </li>
+                        <li>or run <code>$ pip install flare-capa</code></li>
+                    </ul>
+                </li>
+                <li class="mb-2">
+                    Analyze a sample and save the JSON results:
+                    <ul>
+                        <li><code>$ capa -j /path/to/file > result.json</code></li>
+                    </ul>
+                </li>
+                <li>Load the JSON results file into capa Explorer Web</li>
             </ol>
             <p class="text-700 text-sm m-0">
                 For more detailed information, explore the
                 <a
-                    href="https://github.com/mandiant/capa"
+                    href="https://github.com/mandiant/capa/tree/master/web/explorer"
                     target="_blank"
                     rel="noopener noreferrer"
                     class="text-blue-600 hover:text-blue-800 transition-colors duration-200"
```

Comment:
```suggestion
                    class="underline"
```
for consistency with the two other links

Reply:
also remove this line

https://github.com/mandiant/capa/blob/1a467455ba7cd938cd323dc214a342f5a8497880/web/explorer/src/components/DescriptionPanel.vue#L66

<details><summary>so the underlines aren't hidden behind </summary>

![grafik](https://github.com/user-attachments/assets/1036cb82-a7ec-4d92-a6bc-51977a021061)

</details>